raiden/transfer/log.py

```diff
@@ -133,10 +149,17 @@ def get_all_state_changes(self):
         )
         return result.fetchall()
 
+    def get_all_state_events(self):
```

Comment:
This is not a good interface, potentially we will have dozens of thousands of events, for a long running node, an interface like this allows for bad code to be written that loads all these events to memory.